src/components/company/CompanyFacts.astro

```diff
@@ -90,10 +90,13 @@ const reportURL =
             <div class="flex flex-wrap items-center justify-between gap-x-4 gap-y-1 text-sm">
               <h3>Omsättning ({year})</h3>
               <p class="text-sm text-orange-250">
-                {company.baseFacts[year].turnover.toLocaleString('sv-SE', {
-                  maximumFractionDigits: 0,
-                }) +
-                  ' ' +
+                {(company.baseFacts[year].turnover / 1000_000).toLocaleString(
+                  'sv-SE',
+                  {
+                    maximumFractionDigits: 0,
+                  },
+                ) +
+                  ' M' +
```

Comment:
I tried implementing this change in another branch so we could release this, but the turnover data is not accurate enough. Some companies seem to get a value in `MSEK` but the unit is still reported as `SEK`. Others seem to have `SEK`, and others seem to have `kSEK`, when comparing agianst their actual turnover numbers I could find in other places.

We definitely need to format turnover properly, but it's important we have consistent data to make it work well.

Let's follow this up in a separate issue: #179

Reply:
Ok, I will remove this from the pull request (it was not intentional to include into this PR)